subsys/bluetooth/host/smp.c

```diff
@@ -404,6 +404,33 @@ static enum bt_security_err auth_err_get(u8_t smp_err)
 	}
 }
 
+#if defined(BT_SMP_PAIRING_ALLOW)
```

Comment:
It's `CONFIG_BT_SMP_PAIRING_ALLOW`, not `BT_SMP_PAIRING_ALLOW`, i.e. it seems you never tested or even compiled this piece of code.

Reply:
OMG, good catch. The flag name was changed last minute, but the same flag was used in the tests, so they did run, but the code was never called after that. Thanks!